src/turbopuffer.test.ts

```diff
@@ -245,6 +245,21 @@ test("bm25_with_default_schema_and_simple_query", async () => {
   expect(results[0].id).toEqual(2);
 });
 
+test("namespaces", async () => {
```

Comment:
because of the query params that are being passed, this will also test the URL param building